src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+
+container?.addEventListener('click', (ev) => {
+  const table = document.querySelector('.field');
+  const btAppendRow = container.querySelector('.append-row');
+  const btDelRow = container.querySelector('.remove-row');
+  const btAppendCol = container.querySelector('.append-column');
+  const btDelCol = container.querySelector('.remove-column');
+
+  switch (ev.target.classList[0]) {
```

Comment:
here is problem, why on demo link not works remove buttons.
looks like brawser build classnames by abc.
try to use classList.contains() method.
<img width="465" alt="Screenshot 2024-07-10 at 06 49 47" src="https://github.com/mate-academy/js_growth_table_DOM/assets/33029864/f710d30b-330c-41d8-8c90-9dc0b4eb33e7">

Reply:
Interestingly in HTML button classes were always last, and it passed on my machine. I guess the indexing of the classes is not a reliable way to impliment...